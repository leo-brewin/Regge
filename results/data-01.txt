# defect
-1.228708139395745760E-01

# i  j   d(defect)/d(lsq(i,j))
  1  2  9.641602486002307000E+01
  1  3  9.614613406421299200E+01
  2  3  9.614613406421300000E+01
  4  1 -1.296909212793604250E+02
  4  2 -1.296909212793604360E+02
  4  3 -1.293325447210422110E+02
  5  1 -3.581082002604849590E+00
  5  2 -3.581082002604849600E+00
  5  3 -3.572519469818212440E+00
  4  5  2.273127596926181170E+01
  6  1 -3.705685385888513400E+00
  6  2 -3.705685385888513430E+00
  6  3 -3.685050916045961240E+00
  5  6  2.285244332101408850E+01
  7  1 -3.578135836611291410E+00
  7  2 -3.578135836611291420E+00
  7  3 -3.569198517400795380E+00
  6  7  2.284525382277863230E+01
  8  1 -3.543093132643915720E+00
  8  2 -3.543093132643915670E+00
  8  3 -3.550696775484530410E+00
  7  8  2.272789855638649750E+01
  9  1 -1.448792661471501300E+02
  9  2 -1.448792661471501430E+02
  9  3 -1.444592438666566680E+02
  8  9  2.278562926876741870E+01
  9  4  2.036794252668566610E+02
