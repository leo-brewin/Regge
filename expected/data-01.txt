# defect
-1.228708139395747190E-01

# i  j   d(defect)/d(lsq(i,j))
  1  2  9.641602486002245830E+01
  1  3  9.614613406421236670E+01
  2  3  9.614613406421238060E+01
  4  1 -1.296909212793595620E+02
  4  2 -1.296909212793595820E+02
  4  3 -1.293325447210413390E+02
  5  1 -3.581082002604849620E+00
  5  2 -3.581082002604849660E+00
  5  3 -3.572519469818212500E+00
  4  5  2.273127596926181140E+01
  6  1 -3.705685385888513450E+00
  6  2 -3.705685385888513400E+00
  6  3 -3.685050916045961190E+00
  5  6  2.285244332101408860E+01
  7  1 -3.578135836611291430E+00
  7  2 -3.578135836611291410E+00
  7  3 -3.569198517400795390E+00
  6  7  2.284525382277863240E+01
  8  1 -3.543093132643915700E+00
  8  2 -3.543093132643915700E+00
  8  3 -3.550696775484530430E+00
  7  8  2.272789855638649750E+01
  9  1 -1.448792661471491370E+02
  9  2 -1.448792661471491590E+02
  9  3 -1.444592438666556630E+02
  8  9  2.278562926876741890E+01
  9  4  2.036794252668552750E+02
